### PR TITLE
main/libzdb: upgrade to 3.2

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: main/libzdb/APKBUILD [hunk 1]
```diff
@@ -27,13 +27,13 @@ build() {
 
 check() {
 	cd "$builddir"
-	make verify
+	yes . | make verify
```
Comment (tcely) [inline main/libzdb/APKBUILD]: Adding a comment about what `yes .` is doing here would be helpful.

Comment (maxice8) [reply main/libzdb/APKBUILD]: Added a lengthy explanation

**Comment by tcely** [comment]
@maxice8 
This doesn't appear to be used by any other packages. If you want to become maintainer I'd suggest moving this to `community`, otherwise it will likely be moved to `unmaintained`.

**Comment by maxice8** [comment]
On Tue, May 07, 2019 at 11:10:26AM +0000, tcely wrote:
> @maxice8 
> This doesn't appear to be used by any other packages. If you want to become maintainer I'd suggest moving this to `community`, otherwise it will likely be moved to `unmaintained`.
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/7324#issuecomment-490035324

Sounds good, i'm adding a commit to move it to unmaintained

**Comment by tcely** [comment]
Merged in https://github.com/alpinelinux/aports/commit/d65de9b8864c49b84deb2f1b9a3352995d7fe696 and https://github.com/alpinelinux/aports/commit/cc71cfc7c67810e2a80b74559f7e50274eeb1183 by @ncopa.

**Pull request closed without merging**